src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/auth/restrict/ClientAccessControlService.java

```diff
@@ -46,6 +46,20 @@ public boolean hasPrivilegedAccess(String serviceAuthToken, AccessControlRespons
         return privilegedAccessClients.contains(serviceName);
     }
 
+    /**
+     * Extracts client id from service authorization token and returns if client is whitelisted as exclusiveClient.
```

Comment:
Comment should not mention exclusive client as it's confusing.

Reply:
@priyankaVerma21 - Have you addressed this one?

Reply:
It has been resolved now . Thank you